src/types/projects.ts

```diff
@@ -187,30 +187,29 @@ export interface JobDefinition {
 }
 
 export interface ListProject {
-  id: string
-  ext_id: string
-  reference_number: string
-  institution_id: string
-  type_classifier_value_id: string
+  id?: string
```

Comment:
**[blocker]** (Possibly)

Some or a lot of these props should always exist for a project (for example id should always exist). It is better to have types that are strict, but what was the use case which required the ? here, so the use case, where some of these fields are missing for a ListProject type object ?